pom.xml

```diff
@@ -26,38 +26,17 @@
     <description>dockstore cli parent POM</description>
     <url>https://github.com/dockstore/dockstore-cli</url>
 
+    <!--
+    The following properties are mostly used in the plugins section
+    as the versions for the dependencies elements are provided
+    in the dockstore bom-internal pom.xml (BOM), unless specifically overridden
+    in dependencies elements
+    -->
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
         <github.url>scm:git:git@github.com:dockstore/dockstore-cli.git</github.url>
-        <dropwizard.version>1.3.17</dropwizard.version>
-        <dropwizard-annotations.version>4.0.5</dropwizard-annotations.version>
-        <swagger-annotations-version>1.6.0</swagger-annotations-version>
-        <openapi-annotations-version>2.1.1</openapi-annotations-version>
         <dockstore-core.version>1.9.0</dockstore-core.version>
-        <assertj.version>3.9.1</assertj.version>
-        <jersey-version>2.25.1</jersey-version>
-        <junit-version>4.13</junit-version>
-        <logback.version>1.2.3</logback.version>
-        <aws.version>2.5.25</aws.version>
-        <powermock.version>2.0.4</powermock.version>
-        <postgresql.version>42.2.5</postgresql.version>
-        <mockito.version>2.28.2</mockito.version>
-        <cwlavro.version>2.0.4.4</cwlavro.version>
-        <okhttp.version>3.14.2</okhttp.version>
-        <slf4j.version>1.7.26</slf4j.version>
-        <swagger-ui.version>3.20.5</swagger-ui.version>
-        <maven-surefire.version>2.21.0</maven-surefire.version>
         <maven-failsafe.version>2.21.0</maven-failsafe.version>
-        <httpcomponents.version>4.5.10</httpcomponents.version>
-        <httpcore.version>4.4.12</httpcore.version>
-        <jetty-servlet.version>9.4.18.v20190429</jetty-servlet.version>
```

Comment:
Question for all, how come there's no warning like the web service? Also, shouldn't the CLI use a version that does not have a warning? https://github.com/dockstore/dockstore/network/alert/bom-internal/pom.xml/org.eclipse.jetty:jetty-server/open

Reply:
I don't know why the (dependabot?) warnings would be different.
I agree that we should use a version that solves the CVE.
(might need to be more than one PR)

Reply:
Looks like we don't have dependabot set up for this repo? that or I don't have permissions to see it

Reply:
FYI, added garyluu to permissions for dependabot